src/styles.css

```diff
@@ -0,0 +1,284 @@
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.container-login button {
+    border: none;
+    border-radius: 3px;
+    padding: 0.5rem;
+    margin-bottom: 1rem;
+}
+
+.container-login button:focus {
+    outline: none;
+    cursor: pointer;
+}
+
+/* HEADER */
+.container-header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 1rem 3rem;
+    background-color: var(--color-secondary-black);   
+    height: 10vh;
+    margin: 0;
+}
+
+.header-hamburger {
+    display: none;
+}
+
+.header-brand {
+    display: block;
+}
+
+.header-form {
+    flex: 0.6;
+    display: flex;
+    padding: 0.1rem;
+    margin: 0.1rem;
+    border-radius: 2px;
+    border: 1px solid var(--color-border);
+    background-color: var(--color-secondary-black);
+}
+
+.header-form input {
+    width: 100%;
+    border: none;
+    background: transparent;
+    font-size: 0.9em;
+    font-weight: bold;
+    padding: 0.8rem 0.5rem;
+    color: var(--color-text);
+}
+
+.header-form input:focus {
+    outline: none;
+}
+
+.header-form button {
+    padding: 0 1.25rem;
+    color: var(--color-text);
+    background: transparent;
+    border: none; 
+}
+
+.header-avatar {
+    border-radius: 50%;
+    height: 40px;
+    width: 40px;
+}
+
+@media screen and (max-width: 600px) {
+    .container-header {
+        justify-content: space-evenly;
+        padding: 1rem 1rem;
+    }
+    .header-hamburger {
+        display: block;
+    }
+
+    .header-form {
+        flex: 1;
+        margin: 0 1rem;
+    }
+
+    .header-brand {
+        display: none;
+    }
+}
+
+/* SIDEBAR */
+.sidebar {
+    background-color: var(--color-secondary-black);
+    display: flex;
+    flex-direction: column;
+    width: 200px;
+    height: 90vh;
+    padding-top: 1rem;
+}
+
+.sidebar .nav-active {
+    display: flex;
+    align-items: center;
+    padding: 0.5rem 1.5rem;
+    margin: 0.2rem 0;
+    cursor: pointer;
+    color: var(--color-text);
+    text-decoration: none;
+}
+
+.nav-active span {
+    margin-left: 0.5rem;   
+    font-size: 0.9em; 
+    cursor: pointer;
+    color: var(--color-text);
+    text-decoration: none;
+}
+
+.sidebar li {
+    display: flex;
+    align-items: center;
+    padding: 0.5rem 1.5rem;
+    margin: 0.2rem 0;
+    cursor: pointer;
+}
+
+.sidebar li>span {
+    margin-left: 0.5rem;   
+    font-size: 0.9em; 
+}
+
+.sidebar li:hover {
+    background-color: var(--color-border);
+}
+
+.sidebar hr {
+    background-color: var(--color-border);
+    margin: 1rem 0;
+}
+.sidebar.open {
+    transform: translateX(0);
+}
+
+@media screen and (max-width: 1000px) {
+    .sidebar {
+        width: 90px;
+    }
+    .sidebar li {
+        justify-content: center;
+    }
+    .sidebar li>span {
+        display: none;
+    }    
+}
+
+@media screen and (max-width: 600px) {
+    .sidebar {
+        transform: translateX(-120%);
+        position: fixed;
+        z-index: 999;
```

Comment:
add this z-index too to your variables list

Reply:
okay!